### PR TITLE
feat: add translations of `es_419` and `es_ES`

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR adds translations of `es_419` and `es_ES`  to the H5P XBlock.

### How To Test
1. Install this branch in your environment.
2. Change the platform language in `(Account → Site Preferences → Site Language)`. Set it to "Spanish (Latin America)"
3. Add a H5P block in a unit from Studio.
4. You should see the block in the selected language.

### Using translations commands

A `Makefile` was created that includes the necessary commands [(based on the edX cookie-cutter)](https://github.com/openedx/edx-cookiecutters/blob/master/cookiecutter-xblock/%7B%7Bcookiecutter.repo_name%7D%7D/Makefile) to extract and compile the translations.

1. Run `make extract_translations`, this will update the strings to be translated in `h5pxblock/locale/en/LC_MESSAGES/text.po`
2. Create a folder for your translations (If it does not exist) in `h5pxblock/locale/`, ie: `h5pxblock/locale/es_419/LC_MESSAGES/`, and create your `text.po` file, based on the file generated above.
3. Add all translations for strings according to language.
4. Run `make compile_translations`, this will generate the `text.mo` file in all created language folders.

### Screenshots

1. In the LMS
image
2. In Studio
image
image
image
image

### CODE REVIEW

File: h5pxblock/h5pxblock.py [hunk 1]
```diff
@@ -116,9 +118,11 @@ def resource_string(self, path):
         return data.decode("utf8")
 
     def render_template(self, template_path, context):
-        template_str = self.resource_string(template_path)
-        template = Template(template_str)
-        return template.render(Context(context))
+        return loader.render_django_template(
```
Comment (mariajgrimaldi) [inline h5pxblock/h5pxblock.py:132]: Can you add docstrings here? thanks!

**Review by mariajgrimaldi — APPROVED** [review]
This is deployed in one of our instances and it's working fine! Thanks

**Comment by ziafazal** [comment]
@BryanttV thanks for creating this PR. However, it appears E2E tests are failing on it.

**Comment by mariajgrimaldi** [comment]
@ziafazal: I added xblock-utils that seem missing in the testing environment. Can we rerun the tests?

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**